nipype/interfaces/fsl/ICA_AROMA.py

```diff
@@ -0,0 +1,115 @@
+#ICA_AROMA pulled from: https://github.com/rhr-pruim/ICA-AROMA
+#This assumes ICA_AROMA.py is already installed and callable via $PATH
+from nipype.interfaces.base import (
+    TraitedSpec,
+    CommandLineInputSpec,
+    CommandLine,
+    File,
+    Directory,
+    traits,
+    OutputMultiPath
+)
+import os
+
+class ICA_AROMAInputSpec(CommandLineInputSpec):
+    featDir = Directory(exists=True,
+                        desc='If a feat directory exists and temporal filtering '
+                        'has not been run yet, ICA_AROMA can use the files in '
+                        'this directory.',mandatory=False,xor=['infile','mask','affmat','warp','mc'])
+    infile = File(exists=True, 
+                  desc='volume to be denoised', 
+                  argstr='-i %s',mandatory=False,xor=['featDir'])
+    outdir = Directory(desc='path to output directory', 
+                  argstr='-o %s',mandatory=True)
+    mask = File(exists=True, 
+                desc='path/name volume mask', 
+                argstr='-m %s',mandatory=False,xor=['featDir'])
+    dim = traits.Int(desc='Dimensionality reduction when running '
+                     'MELODIC (defualt is automatic estimation)', 
+                     argstr='-dim %d')
+    TR = traits.Float(desc='TR in seconds. If this is not specified '
+                      'the TR will be extracted from the header of the fMRI nifti file.',
+                      argstr='%.2f')
+    melodic_dir = Directory(exists=True,
+                            desc='path to MELODIC directory if MELODIC has already been ran',
+                            argstr='-meldir %s')
+    affmat = File(exists=True,
+                  desc='path/name of the mat-file describing the '
+                  'affine registration (e.g. FSL FLIRT) of the '
+                  'functional data to structural space (.mat file)',
+                  argstr='-affmat %s',mandatory=False,xor=['featDir'])
+    warp = File(exists=True,
```

Comment:
I'd prefer `warp_file`. And possibly `fnirt_warp_file`, since each packages has its own warp format.